README.md

```diff
@@ -63,41 +131,109 @@ systemctl status multipathd
 multipath -ll
 ```
 
-**Important:** When configuring Linux multipath devices, verify that the `find_multipaths` parameter in the `multipath.conf` file is disabled. In RHEL 7.x, remove the`find_multipaths yes` string from the `multipath.conf` file.
+##### 2.2 Configuring for Kubernetes users (RHEL)
+Create and set the relevant storage system parameters in the `/etc/multipath.conf` file. You can also use the default `multipath.conf` file, located in the `/usr/share/doc/device-mapper-multipath-*` directory.
+
+Verify that the `systemctl status multipathd` output indicates that the multipath status is active and error-free.
+
+```bash
+yum install device-mapper-multipath
+modprobe dm-multipath
+systemctl enable multipathd
+systemctl start multipathd
+systemctl status multipathd
+multipath -ll
+```
 
 #### 3. Configure storage system connectivity
-3.1. Define the hostname of each Kubernetes node on the relevant storage systems with the valid WWPN (for Fibre Channel) or IQN (for iSCSI) of the node. 
+3.1. Define the hostname of each Kubernetes node on the relevant storage systems with the valid WWPN(for Fibre Channel) or IQN(for iSCSI) of the node.
 
 3.2. For Fibre Channel, configure the relevant zoning from the storage to the host.
 
-3.3. For iSCSI, perform the following steps:
 
-3.3.1. Make sure that the login to the iSCSI targets is permanent and remains available after a reboot of the worker node. To do this, verify that the node.startup in the /etc/iscsi/iscsid.conf file is set to automatic. If not, set it as required and then restart the iscsid service `$> service iscsid restart`.
+<br/>
+<br/>
+<br/>
 
-End of worker node setup.
 
+## Installation
 
+### Install the operator
 
 
-<br/>
-<br/>
-<br/>
+#### 1. Download the manifest from GitHub.
 
+```bash
+curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/master/deploy/installer/generated/ibm-block-csi-operator.yaml > ibm-block-csi-operator.yaml
+```
+
+#### 2. (Optional): If required, update the image fields in the ibm-block-csi-operator.yaml.
+
+
+#### 3. Create a namespace.
+
+```bash
+$ kubectl create ns <namespace>
+```
+
+#### 4. Install the operator, while using a user-defined namespace.
+
+```bash
+$ kubectl -n <namespace> apply -f ibm-block-csi-operator.yaml
+```
+
+### Verify the operator is running:
```

Comment:
@ArbelNathan I think this `###` should be something like `#####` - what do you think? and should we fix it soon?